packages/terra-modal-manager/src/ModalManager.jsx

```diff
@@ -49,6 +49,16 @@ const widthFromSize = {
 };
 
 class ModalManager extends React.Component {
+  static checkIsExpandable() {
+    if (document.querySelector(['[data-terra-abstract-modal]'])) {
+      const { height, width } = document.querySelector(['[data-terra-abstract-modal]']).getBoundingClientRect();
+      if (window.innerHeight === height + 20 && window.innerWidth === width + 20) {
```

Comment:
What breakpoint is this? Should this be on certain terra breakpoints?

Reply:
The AbstractModal has a maximum size of 100% - 20px for its height and width. This is checking to see if the modal has reached that size. 

But I'm not quite sure how this would work, as at the time this check occurs, I don't believe the AbstractModal has even mounted yet? Maybe we're getting subsequent renders that make this work appropriately. Ideally we'd be able to cross-reference the current active breakpoint with the current modal size and determine whether or not maximize is possible even before rendering, but that logic would get pretty complicated.

Reply:
I think we can use [dimensions / size values that are passed to modal in `renderModal()`](https://github.com/cerner/terra-framework/blob/580b034bbe0f225b58e20c33619844af2e3cff3e/packages/terra-modal-manager/src/ModalManager.jsx#L84-L86) instead of using `data-terra-abstract-modal` height and width for calculation to hide maximize on header.
```jsx
For Dimensions :
 shouldHideMaximize = (windowHeight <= manager.disclosure.dimensions.height + 20 && windowWidth <= manager.disclosure.dimensions.width + 20);

For Size : 
 shouldHideMaximize = (windowHeight <= heightFromSize[manager.disclosure.size] + 20 && windowWidth <= widthFromSize[manager.disclosure.size] + 20);
```

Reply:
Updated 7f5a84c